readthedocs/docsitalia/settings/docker.py

```diff
@@ -149,6 +149,15 @@ def INSTALLED_APPS(self):  # noqa
             'hosts': ES_HOSTS
         },
     }
+
+    @property
+    def ES_INDEXES(self):  # noqa - avoid pep8 N802
+        es_indexes = super(DocsItaliaDockerSettings, self).ES_INDEXES
```

Comment:
Qui userei `super().ES_INDEXES` come già in altri punti di questo file.

Reply:
:+1: aggiornato